project/cmake/installdata/addon-bindings.txt

```diff
@@ -31,3 +32,6 @@ xbmc/addons/kodi-addon-dev-kit/include/kodi/libKODI_inputstream.h
 xbmc/addons/kodi-addon-dev-kit/include/kodi/libKODI_peripheral.h
 xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_pvr.h
 xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_codec.h
+xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxPacket.h
+xbmc/cores/AudioEngine/Utils/AEChannelData.h
+xbmc/filesystem/IFileTypes.h
```

Comment:
None were forgotten as such. They did not exist / had not been promoted to addon bindings when i sent the initial version.